orbit-db-cli/.github/workflows/automerge.yml

```diff
@@ -0,0 +1,10 @@
+name: automerge
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '17 2,5,8,11 * * *'
+
+jobs:
+  automerge:
+    uses: oclif/github-workflows/.github/workflows/automerge.yml@main
+    secrets: inherit
```

Comment:
what is this action used for? DO we need it?

Reply:
we don't, this was an oclif default action that was automatically created